### ISSUE TITLE
Plotting in Cutadapt function breaks the code even if LDist is set to False

### ISSUE THREAD

**Issue by DominikBuchner** [VascoElbrecht/JAMP#23]
Hi Vasco,
in some rare cases the Cutadapt function will run into an infinite xlim error, even though the data handling completed successfully. Got it worked around by commenting out the plotting part in the sourcecode and write a placeholder to the log.
Suggested solution: If LDist is set to False, avoid all plots, not just the length distribution plots. Or even better add another argument to seperate both plotting options.
best Dominik

**Comment by VascoElbrecht** [comment]
yes, thats an issue with NA values, I will work on a solution =) thank you for the reporting
